.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 
 install:
   - ./pkg-build.sh install_deps
-  - ./pkg-build.sh install_github jimhester/covr
+  - ./pkg-build.sh install_github jimhester/covr hadley/testthat#443
```

Comment:
OK, the testthat PR was merged, and as I see a new testthat version will go to CRAN soon. So we can merge this now.